src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -481,7 +481,9 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
         .map((cells) => {
           return cells
             .map((c) =>
-              c.cell ? this.getters.getCellText(c.position, this.getters.shouldShowFormulas()) : ""
+              this.getters.shouldShowFormulas()
```

Comment:
```suggestion
              this.getters.shouldShowFormulas() && cell?.isFormula
``` 
you should preserve the same logic as `getCellText`. With the current suggestion, if I were to **CUT** a cell with `{content: 1, format: "mm/dd/yyyy"}`  while `this.getters.shouldShowFormulas() === true` I would end up with  `1` in my clipboard, whereas other spreadsheets would yield `12/31/1899`.

I think a test would be welcome here to make sure we don't break tahat behaviour in the future :)

Reply:
I don't think I see a test for this